themes/ee/asset/javascript/src/cp/form_validation.js

```diff
@@ -326,9 +326,14 @@ EE.cp.formValidation = {
 			grid = true;
 		}
 
+		if (fieldset.find('.fluid').size()>0)
+		{
+			container = field.parents('td');
+			grid = true;
```

Comment:
I did not test this, but it does not seem to be that the sole condition `fieldset.find('.fluid').size()` is enough to assume that `grid = true;`